pkg/scaffold/gopkgtoml.go

```diff
@@ -58,6 +58,10 @@ required = [
   # revision for tag "kubernetes-1.12.3"
   revision = "3dcf91f64f638563e5106f21f50c31fa361c918d"
 
+[[override]]
+  name = "k8s.io/kube-openapi"
+  branch = "master"
```

Comment:
Do we really need master? 🤔 Also asking because in my https://github.com/operator-framework/operator-sdk/pull/786/files#diff-836546cc53507f6b2d581088903b1785R31 I require that :)

Reply:
Nope, I can lock to `0cf8f7e6ed1d2e3d47d02e3b6e559369af24d803`.

Reply:
Thanks! 👍